src/main/kotlin/org/jitsi/jibri/sipgateway/pjsua/PjsuaClient.kt

```diff
@@ -92,14 +115,22 @@ class PjsuaClient(
             // The proxy we'll use for all the outgoing SIP requests;
             // The client should not specify a Route header in the sip INVITE message. Using hide will let the server set the Route header
             if (pjsuaClientParams.sipClientParams.userName != null) {
-                command.add("--proxy=sip:${pjsuaClientParams.sipClientParams.userName.substringAfter('@')};" +
-                        "transport=tcp;hide")
+                command.add(
+                    "--proxy=$sipScheme:${pjsuaClientParams.sipClientParams.userName.substringAfter('@')};" +
+                            "transport=tcp;hide"
+                )
+            }
+            if (!sipOutboundPrefix.isNullOrEmpty()) {
+                sipAddress = sipOutboundPrefix.plus(sipAddress)
```

Comment:
Ah, I see this is a bit tricky for sipAddress since it has to also have the outbound prefix appended, you could do something like:

```
if (sipOutboundPrefix.isNUllOrEmpty() {
    command.add("$sipScheme:$sipAddress$sipOutboundPrefix")
} else {
    command.add("$sipScheme:$sipAddress")
}
```